compiler/tests-common/tests/org/jetbrains/kotlin/codegen/ir/AbstractFirBlackBoxAgainstJavaCodegenTest.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2010-2019 JetBrains s.r.o. and Kotlin Programming Language contributors.
```

Comment:
Out of curiosity, is this auto generated? I wonder it should be 2020 or somewhere in the generator is using an old one?

Reply:
I think it was just copied from `AbstractFirBlackBoxCodegenTest.kt`. "New File" generates correct comment, at least on my PC.

Reply:
Yes, you caught me copy-pasting. 😁